search_params.py

```diff
@@ -231,7 +232,7 @@ def retrain_best_model(exp_name, best_config, best_log_dir, retrain):
     with open(os.path.join(checkpoint_dir, "params.yml"), "w") as fp:
         yaml.dump(dict(best_config), fp)
 
-    data = load_static_data(best_config, merge_train_val=best_config.merge_train_val)
+    data = load_static_data(best_config)
```

Comment:
Same as the previous comment. Why code changes here?

Reply:
Not really...But the argument has already be included in config